src/main/java/me/mrCookieSlime/Slimefun/Events/DisenchanterEvent.java

```diff
@@ -0,0 +1,42 @@
+package me.mrCookieSlime.Slimefun.Events;
+
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.bukkit.inventory.ItemStack;
+
+public class DisenchanterEvent extends Event implements Cancellable {
+	
+	private static final HandlerList handlers = new HandlerList();
+
+	private final ItemStack sfItem;
+	private boolean cancelled;
+
+	public HandlerList getHandlers() {
+	    return handlers;
+	}
+	
+	public static HandlerList getHandlerList() {
+        return handlers;
+    }
+	
+	public DisenchanterEvent(ItemStack sfitem){
+		super(true);
+		this.sfItem = sfitem;
+	}
+	
+	public ItemStack getSfItem() {
```

Comment:
Don't use abbreviations in the method name here, just do getItem()
Especially because this is not a SlimefunItem, it just an ItemStack.